src/components/Perils/index.tsx

```diff
@@ -0,0 +1,41 @@
+import axios from 'axios'
+import React, { useEffect, useState } from 'react'
+import { PerilCollection } from './PerilCollection/PerilCollection'
+import { Peril, TypeOfContract } from './types'
+
+interface Props {
+  insuranceType: TypeOfContract
+}
+
+export const Perils: React.FC<Props> = ({ insuranceType }) => {
+  const [perils, setPerils] = useState<[] | Peril[]>([])
+
+  useEffect(() => {
+    const fetchPerils = async () => {
+      const url = `https://graphql.dev.hedvigit.com/graphql`
+      const data = `{"operationName":"Perils","variables":{"typeOfContract":"${insuranceType}","locale":"sv_SE"},"query":"query Perils($typeOfContract: TypeOfContract!, $locale: Locale!) {\\n perils(contractType: $typeOfContract, locale: $locale) {\\n title\\n description\\n covered\\n icon {\\n variants {\\n light {\\n svgUrl\\n }\\n }\\n }\\n }\\n}\\n"}`
```

Comment:
could we maybe use `JSON.stringify` and do the actual query in a template string intsead? Imagine coming back to this in a few months and trying to make out what we're querying.

So this instead becomes:
```js
const data = JSON.stringify(
{
    "operationName": "Perils",
    "variables": {
        "typeOfContract": insuranceType,
        "locale": "sv_SE"
    },
    "query": `
        query Perils($typeOfContract: TypeOfContract!, $locale: Locale!) {
          perils(contractType: $typeOfContract, locale: $locale) {
            title
            description
            covered
            icon {
              variants {
                light {
                  svgUrl
                }
              }
            }
          }
        }
`
}
)
```

Reply:
Acutally, i think you don't even need to stringify it with axios. it should be sufficient to pass the object directly, and axios will covert it to JSON automatically, if i remember correctly